docs/Home.md

```diff
@@ -2,41 +2,77 @@
 
-Quarkiverse provides repository hosting (including build, CI and release publishing setup) for Quarkus extension projects contributed by the community.
-Quarkus extensions hosted in the Quarkiverse organization will automatically be included into the Quarkus extensions catalog displayed on [code.quarkus.io](http://code.quarkus.io)
-and the Quarkus command line tools (such as `mvn quarkus:list-extensions`, `gradle listExtensions`).
+Quarkus is a Kubernetes Native Java stack tailored for OpenJDK HotSpot and GraalVM, crafted from the best of breed Java libraries and standards, and it is also a ecosystem of ever growing extensions. In the beginning we were adding all extensions to the core Quarkus repository but eventually it became to much of overhead or just not always sensisble to include an extension into the core. 
+
+Some have since then created their own repositories external to Quarks, but others just never happeend because not everyone wants the overhead of maintaining a build setup while in other cases we actually had an idea for an extension but not solidified enough yet to be included in core.
+
+Quarkiverse is our proposed solution to create a "home" for such extensions/projects.
+
+#What is Quarkiverse
+
+This Quarkiverse github organization provides repository hosting (including build, CI and release publishing setup) for Quarkus extension projects contributed by the community.
+
+Quarkus extensions hosted in the Quarkiverse organization will by default be included into the Quarkus extensions catalog displayed on [code.quarkus.io](http://code.quarkus.io) and the Quarkus command line tools (such as `mvn quarkus:list-extensions`, `gradle listExtensions`). To stay listed only requirement
+is that the extension keeps functioniong, stays uptodate and cause no harm.
 
 # Why Quarkiverse
 
 In the early days and actually up until recently, Quarkus extensions contributed by the community members (including the core Quarkus team) have been welcome in the Quarkus core repository at https://github.com/quarkusio/quarkus. Today the core repository has grown a lot and it's evident that it simply becomes troublesome to work with it for its contributors. That and other maintenance and infrastructure related issues inspired the creation of a separate organization called Quarkiverse to host Quarkus extension projects contributed by the community (including the Quarkus core team members themselves).
 
+The advantages to join Quarkiverse are:
+
+  . Automated and secured publishing of your maven releases to Maven Central.
+  . Automated Cross-testing of your extension with Quarkus builds/releases (see https://github.com/quarkusio/quarkus-ecosystem-ci)
+  . Inclusion in registry used by Quarkus tooling to browse extensions (i.e. code.quarkus, command line and IDE tools).
+  . Consistent formatting and release of project
+  . Quarkus team members can help and fix issues if project has build issues.
```

Comment:
There is no such promise though. Perhaps, we shouldn't advertise that. That could create a false expectation.

Reply:
I agree, at the bare minimum we should say that it would have higher visibility, so community members can contribute too

Reply:
I've added a "in emergency" - i'm not listing it here as a promise it will happen but to be transparent that by being in here we have the possiblity to do fixes in case maintainer is missing-in-action.